quality-control/human-clinical-and-health-data.md

```diff
@@ -1,27 +1,79 @@
 ---
 title: Human clinical and health data
-description: Tools and methods to assess quality.
-contributors: []
-no_robots: true
-search_exclude: true
-sitemap: false
+description: General considerations for human clinical and health data quality control.
+contributors: [Vladimira Koudelakova, Eva Garcia Alvarez, Maria A. Rujano, Maria Panagiotopoulou, Bert Droesbeke]
 page_id: hchd_quality_control
 redirect_from: /human-clinical-and-health-data/quality-control
-rdmkit:
-  - name:
-    url:
-training:
-  - name:
-    registry:
-    url:
-# More information on how to fill in this metadata section can be found here https://www.infectious-diseases-toolkit.org/contribute/page-metadata
 ---
 
-**We are still working on the content for this page.** If you are interested in adding to the page, then:
+## Introduction
+Human clinical and health data are collected at information care systems. Therefore, quality measures are needed in order to avoid as many errors as possible when collecting and processing these data, keeping it reliable. In addition, different information collection systems are used in different health centres, so quality measures should also take interoperability into account. Here we present several considerations, existing approaches, and frameworks for managing quality in clinical and health data.
```

Comment:
I suggest to change information care system -> health information system

Reply:
changed